src/components/biggive-accordion/biggive-accordion.tsx

```diff
@@ -0,0 +1,59 @@
+import { Component, Prop, Element, h } from '@stencil/core';
+import { brandColour } from '../../globals/brand-colour';
+import { spacingOption } from '../../globals/spacing-option';
+
+@Component({
+  tag: 'biggive-accordion',
+  styleUrl: 'biggive-accordion.scss',
+  shadow: true,
+})
+export class BiggiveAccordion {
+  @Element() host: HTMLBiggiveAccordionElement;
+
+  @Prop() spaceBelow: spacingOption = 0;
+
+  @Prop() textColour: brandColour = 'black';
+
+  @Prop() headingColour: brandColour = 'primary';
+
+  children: Array<any> = [];
```

Comment:
Could we use a more specific type instead of Array of any? I'm not sure if it should be `Array<BiggiveAccordionEntry>` or something else. Changing it to that does create a few compile errors, but using `any` has potential to suppress errors at compile time and let them happen at runtime, possibly in production first if that particular usage isn't always fully tested.

Reply:
Done